docs/ivys.js

```diff
@@ -37,23 +40,44 @@ IvyApp.prototype.addItem = function (imageId, imagePath) {
 IvyApp.prototype.removeItem = function (imageId) {
     // removes an image from masonry grid
     "use strict";
+    var imageElement = document.getElementById(imageId);
     
+    this.$grid.masonry('remove', imageElement);
+    
+    // Re-layout the images on the page
+    $('.grid').imagesLoaded(function () {
+        this.$grid.masonry('layout');
+    }.bind(this));
 };
 
-IvyApp.prototype.addTag = function (tagName) {
-    // Add a tag to the page
+IvyApp.prototype.addTag = function (tagId, tagObject) {
+    // Add a tag to the page by Id
+    // tagObject is the object containing all the tags
```

Comment:
Is there only ever one tagObject? Why not just have it live on the IvyApp instead of passing it around?